tests/test_enum.py

```diff
@@ -66,20 +66,20 @@ def test_unscoped_enum():
     # Unscoped enums will accept ==/!= int comparisons
     y = m.UnscopedEnum.ETwo
     assert y == 2
-    assert 2 == y
+    assert y == 2
```

Comment:
Yes, you can disable checks per file.  This is yoda-conditions.

Reply:
Would it be useful if I jump in and try? — Asking to make sure we're not both working on it at the same time.

Reply:
I can do it, since I'm looking at it now. It's just a simple series of commands:

```bash
$ git checkout master -- tests/test_enum.py
$ pre-commit run -a
...
Fixed 19 errors:
- tests/test_enum.py:
    10 × SIM201 (negate-equal-op)
     7 × SIM300 (yoda-conditions)
     2 × SIM202 (negate-not-equal-op)
$ git restore .
```

Then I add those codes to pyproject.toml.

Reply:
Thanks for the explanation!

In pyproject.toml:
```
"tests/test_enum.py" = ["SIM201", "SIM300", "SIM202"]
```

For someone externally this will be very difficult to discover. Preferred alternative to make this more obvious (tested):

test_embed.py (e.g. top):
```
# It is critical that ruff does not rewrite the tests for overloaded operators:
# ruff: noqa SIM201 SIM202 SIM300
```

Reply:
A) that was just added a release or two ago :)

B) I'm not sure that's the right syntax. Did you verify that this is not a global ignore of all codes + some extra comment letters? The docs say `ruff: noqa: SIM201 SIM202 SIM300` (note the second colon).

Sure, happy to do it that way.

Reply:
Answer to B: yes, that's exactly what it is. If the colon is missing, that's just a comment after a total `noqa`. Hmm, wait a minute, even with the colon, it still seems to be ignoring too much. Will investigate.

Reply:
> Answer to B: yes, that's exactly what it is.

Oh ... sorry, I picked up the syntax from some documentation page and apparently was too happy too quickly. Thanks for looking into this!

Reply:
It does show a warning, but you need a Ruff new enough to support it, which I didn't have. It's (as of writing) the latest version now.